classes/item.rb

```diff
@@ -0,0 +1,39 @@
+require 'date'
+
+class Item
+  attr_accessor :genre, :id, :archived, :author, :publish_date, :label
+
+  def initialize(id, publish_date, archived: false)
+    @id = id || Random.rand(1..1000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def add_genre(genre)
+    @genre = genre
+  end
```

Comment:
- [ ] Kindly add the `self` to the items list as you did in the `add_author` and `add_label` methods.